lib/fetch.js

```diff
@@ -42,7 +30,11 @@ export function sanitizeEventAndNews(item, type) {
     strapline: item.data[`${type}.strapline`] ?
       item.data[`${type}.strapline`].value : null,
     datetime: item.data[`${type}.timestamp`] ?
-      mapDate(item.data[`${type}.timestamp`].value) : null,
+      item.data[`${type}.timestamp`].value : null,
```

Comment:
Or Ramda's `flip' :) Or for more complex re-ordering, [`__`](http://ramdajs.com/0.21.0/docs/#__) is really cool when used in the context of partial application.

`lodash`s [`rearg`](https://lodash.com/docs#rearg) syntax, on the other hand, is a bit verbose I find.